app/src/main/java/travelbook/android/app/data/MemoriesDao.java

```diff
@@ -9,7 +14,55 @@
  */
 public class MemoriesDao {
 
+    private final TravelBookDatabaseOpenHelper travelBookDatabaseOpenHelper;
+    String selection = "is_primary != ?";
+    String[] selectionArgs = {"null"};
```

Comment:
Move `selectionArgs` directly to line 36.
Surround "null" with spaces and capitalize it to point out that it is SQL word: `{ "NULL" }`